packages/perseus/src/renderer.tsx

```diff
@@ -1594,6 +1594,9 @@ class Renderer
         return state;
     };
 
+    /**
+     * Returns an array of widget ids that are empty.
+     */
     emptyWidgets(): ReadonlyArray<string> {
```

Comment:
To confirm my understanding of this one, this is determining if the user hasn't input or selected a value for a scorable widget, correct? 

I wonder if it would be helpful to provide a little explanation about what "empty" means just to make it easier to grok at a quick glance, but this is a totally ignorable nit.

Reply:
Good idea.  I've expanded it.  We can always keep clarifying if it's still not good enough. :)